openshift/sandboxed-containers-operator
# Remove generated files and update docs to generate files as they were

**- Description of the problem which is fixed/What is the use case**
There are files being tracked that are completely generated by make targets.  Fewer files makes maintaining the code easier. 

**- What I did**
Based on the operator-sdk docs [1], I have deleted the generated files from the repo.  One file required an additional command line change to generate the file exactly as it was before deleting.  That change was added to the DEVELOPMENT.md file.  I also add the relevant entries in the .gitignore to keep the files from being added back.

[1] https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/

This fixes: #209 and #208 

**- How to verify it**

If you execute the make file targets all files that were deleted should be recreated exactly the same:

make docker-build
make bundle
make bundle-build CHANNELS=candidate

**- Description for the changelog**

Generated files have been deleted from the repo
